### ISSUE TITLE
Commented out code in the Pre Request script causes failure of the request

### ISSUE THREAD

**Issue by Wyanglade** [usebruno/bruno#3280]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

I have a Patch request that has commented out code in the Pre Request script that returns a successful status (200 OK).

I make a simple change in the Pre Request script (for example, declare a variable that I will not use) and save the change.
When I run the request, it returns a failure (400 Bad request).

If I remove the change I have made to the Pre Request script and run the request, I still have a failure result (400 Bad Request).

If I remove the lines that were commented out, save and run the request again, I have a successful result (200 OK).

The failure is due to the fact that the commented out code is somewhere in the middle of my Pre Request script. Anything after the commented out lines is not executed. It is as if Bruno is not considering that only the commented out lines are as such, but all the rest of the code in the script page is considered as commented out.  The only solution is to remove the lines that are commented out. After saving the change and running the request, we are back to a successful request because all the code in the page is executed again.

### Screenshots/Live demo link

![image](https://github.com/user-attachments/assets/1cc2cf2c-9ccb-4da2-a44a-95a3182031c7)
![image](https://github.com/user-attachments/assets/204f5753-d51f-4beb-9a11-75252b481f4a)
![image](https://github.com/user-attachments/assets/2ea7de38-7450-4667-9bc2-7eecff8308fd)
![image](https://github.com/user-attachments/assets/a8d21e44-f53b-46ab-a78e-43df15d7ba91)

**Comment by Pragadesh-45** [comment]
Hey @Wyanglade,

Thanks for reporting this issue. I wasn't able to reproduce the bug as you described. If possible, could you please provide clear steps or a .bru file to help me replicate it?

**Comment by Wyanglade** [comment]
Thank you @Pragadesh-45 ,

I have attached a .bru file of a Patch request which had commented out code, which did not cause an issue in the Bruno previous version.

**Comment by helloanoop** [comment]
@Wyanglade I see you re-opened the issue. Is this still happening in the latest version of Bruno?

**Comment by Wyanglade** [comment]
I reproduced it at least  at 3 different occasions with different requests.

Basically, I had code that was commented out in a previous version of Bruno, by doing a "Ctrl-/"
The request (Patch request) was successful when executed (returning 200 OK).
I updated to a newer version of Bruno (noticed it last in 1.33.1,  but had noticed it originally in 1.32.1)
I ran the same request and it was returning 400 Bad Request
I cannot share the *.bru file, but the Pre-Request script had commented lines in the middle of it:

![image](https://github.com/user-attachments/assets/f38e1a73-474d-4fca-8aa6-386dd9e7396a)


Any code after the 4 commented out lines did not seem to be executed, as if the rest of the script after the commented out lines were also commented out. Any bru.setVar() variables in the rest of the code that was not commented out were undefined.
I deleted the 4 commented out code without doing any other change in the script and saved.

I ran the request and I have a successful result (200 OK).

It is bizarre that I delete commented out code and I get the expected result.
